lib/faker/phone/pt_pt.ex

```diff
@@ -0,0 +1,73 @@
+defmodule Faker.Phone.PtPt do
+  import Faker, only: [samplerp: 2]
+
+  @moduledoc """
+  Functions for generating phone related data for portugal location
+  """
+
+  @doc """
+  Returns a random phone number.
+
+  ## Examples
+
+      iex> Faker.Phone.PtPt.number()
+      "929999999"
+      iex> Faker.Phone.PtPt.number()
+      "919999999"
+      iex> Faker.Phone.PtPt.number()
+      "234999999"
```

Comment:
Could you please add 4 examples for doctests?

Reply:
done